ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -102,9 +102,13 @@ def initialize(opts = {})
       @oidc_cookie_same_site            = opts.fetch(:oidc_cookie_same_site, @ssl ? 'Off' : 'On')
       @oidc_settings                    = opts.fetch(:oidc_settings, {})
 
-      # CORS setup
+      # API setup
+      @api_uri                          = opts.fetch(:api_cors_uri, "#{@pun_uri}/sys/dashboard/api")
+      # API auth
+      @api_auth_enabled                 = opts.fetch(:api_auth_enabled, false)
+      @api_auth_root                    = opts.fetch(:api_auth_root, "/opt/ood/api")
```

Comment:
This should be `/etc/ood/api` I would think , should not put configuration files of any kind into `/opt`.